### PR TITLE
Google map dev

### PR DESCRIPTION
**Pull request by MarioTrentini**
Hello,

I recently got a GPS watch and have been using your program (thanks by the way).

Please, find some suggestions on the Google Map generation.

Comments are welcomed.

Thanks

### CODE REVIEW

File: src/output/GoogleMap.cc [hunk 1]
```diff
@@ -64,6 +64,30 @@ namespace output
...
+		double avg_speed = session->getAvgSpeed();
+		double min_speed;
+		double max_speed;
+		// let's cap the 10% extreme values to remove samples with
+		// potential artifacts
+		// sort the list of speed for that
+		{
+		  std::list<double> speedList;
+		  int pointCnt=0;
+		  for(auto it = points.begin(); it != points.end(); ++it) {
```
[the remark this answers is not in the document]
Comment (MarioTrentini) [reply src/output/GoogleMap.cc]: OK, thanks, I used a vector and this addresses the segfault at the same time.

**Comment by ColinPitrat** [comment]
OK, the test is still failing but this is now due to your change: the expected output needs to be changed.
I'll handle it.

Thanks for the contribution !
Do not hesitate if you have other ideas.

**Comment by MarioTrentini** [comment]
With my latest changes the integrated tests don't segfault anymore but, they fail since the new html outputs are different from the references.
Should I push a commit updating the references myself ?
By the way, it seems the integration tests are missing a test with laps. I would rather add that first (it would have detected the print of the pointer instead of the lap count).

**Comment by ColinPitrat** [comment]
You can fix the tests and add laps (or add the HTML output in another case that has laps) in another PR if you wish.
Otherwise, I can handle it.

**Comment by MarioTrentini** [comment]
OK, thanks. I have actually other ideas (a friend suggested me to add slope statistics). And I have to push improvements for my watch...

**Pull request merged**